### PR TITLE
Fix lib_deps

### PR DESCRIPTION
**Pull request by eloo**
Hi, here is small PR to make the lib_deps work properly :)

### CODE REVIEW

File: Affenbox/Affenbox.ino [hunk 1]
```diff
@@ -2024,6 +2024,13 @@ static void nextTrack(uint8_t track, bool force /* = false */)
       break;
 
     case AudioBook:
+      numTracksInFolder = mp3.getFolderTrackCount(myFolder->folder);
```
Comment (eloo) [inline Affenbox/Affenbox.ino:2027]: @marco-117 this is the workaround to fix the folderTrackCount issue

**Comment by marco-117** [comment]
Hi eloo, thank you for these PRs. I will merge them as soon i have time to.
At the Moment i build a Test Rack and CI/CD integration for this repo. I have not much time to test all the changes and PRs by hand, so i will automate it.

**Comment by eloo** [comment]
@marco-117 

thanks for the reply and no hurry ;) 
take your time and so far i will work with my own branch :+1: 

getting a CICD pipeline here sounds pretty awesome
maybe you can share a short footage when its finished

best regards

**Comment by marco-117** [comment]
the platformio.ini was updatet to suit the ci/cd purpose.

**Pull request closed without merging**